src/components/footer.js

```diff
@@ -0,0 +1,55 @@
+import React from "react"
+import { makeStyles } from "@material-ui/core"
+import { graphql, useStaticQuery } from "gatsby"
+
+const useStyles = makeStyles(theme => ({
+  footer: {
+    backgroundColor: "#007a7c",
+    color: "white",
+    position: 'relative',
+    bottom: 0,
+    width: '100%'
+  },
+  footerInnerDiv: {
+    margin: `0 auto`,
+    maxWidth: 960,
+    padding: `1.45rem 1.0875rem`,
+    display: `flex`,
+    justifyContent: `space-between`,
+    position: 'sticky'
+  },
+  list: {
+      listStyle: 'none'
+  },
+  footerLink: {
+    color: "white",
+    textDecoration: "none",
+  }
+}))
+
+export default function Footer() {
+  const classes = useStyles()
+
+  const contactInfo = useStaticQuery(graphql`
+    query {
+        contact {
+            email
+            links {
+            twitter
+            }
```

Comment:
There were other contact links that have been added to site (facebook, meetup) that should be included in the footer.